python/test/model_TEST.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# Copyright (C) 2023 Open Source Robotics Foundation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from gz.common import set_verbosity
+from gz_test_deps.sim import TestFixture, Model, World, world_entity
+
+post_iterations = 0
+iterations = 0
+pre_iterations = 0
+
+class TestModel(unittest.TestCase):
+
+    def test_model(self):
+        set_verbosity(4)
+
+        file_path = os.path.dirname(os.path.realpath(__file__))
+        fixture = TestFixture(os.path.join(file_path, 'model_test.sdf'))
+
+        def on_post_udpate_cb(_info, _ecm):
+            global post_iterations
+            post_iterations += 1
+
+        def on_pre_udpate_cb(_info, _ecm):
+            global pre_iterations
+            pre_iterations += 1
+            world_e = world_entity(_ecm)
+            self.assertEqual(1, world_e)
+            w = World(world_e)
+            model = Model(w.model_by_name(_ecm, 'model_test'))
+            # Entity Test
+            self.assertEqual(4, model.entity())
+            # Valid Test
+            self.assertTrue(model.valid(_ecm))
+            # Name Test
+            self.assertEqual('model_test', model.name(_ecm))
+            # Static Test
+            self.assertTrue(model.static(_ecm))
+            # Self Collide Test
+            self.assertTrue(model.self_collide(_ecm))
+            # Wind Mode Test
+            self.assertTrue(model.wind_mode(_ecm))
+            # Get Joints Test
+            self.assertEqual(7, model.joint_by_name(_ecm, 'model_joint_test'))
```

Comment:
Using a hard coded entity ID here is brittle. Future changes in the core library might cause this number to change, so I'd recommend just checking that it's not an invalid entity (`kNullEntity`).

Reply:
Ok this makes sense. I'll do the modifications on all the PRs.

Reply:
Done.